03-Security-In-CICD/Readme.md

```diff
@@ -1,18 +1,23 @@
-# Security Verification Tests (SVTs) in CICD Pipeline
+
+# Security Verification Tests (SVTs)
 
 ![Security_In_CICD](../Images/Security_In_CICD.jpg)
 
 > **Notes** 
 > - We also plan to support miniaturized ARM templates which can be combined with existing templates and, subsequently, to analyze 'ARM template equivalence' from a security standpoint.
```

Comment:
Not sure if we really want to claim on mini arm templates

Reply:
For now I have removed this text.